develop/plone/i18n/contribute_to_translations.rst

```diff
@@ -55,23 +45,21 @@ Here you have the following directories:
 
 - ``locales`` used for core Plone translations.
 - ``locales-addons`` used for some addons packages.
-- ``locales-future`` used for Plone 5 packages. The po files can change
-  really often. The translations will normally be merged to the locales
-  directory when Plone 5 will be released. This directory is not included in
-  ``plone.app.locales`` 4.x releases. For developers: if you want to add a
-  Plone 5 package to this directory, you can add it to the ``plone5``
-  variable in ``experimental/i18n.cfg``, don't forget to add a line in
-  ``[sources]`` if it's missing.
+- ``locales-future`` used for Plone 5 packages.
```

Comment:
I thought I fixed this documentation already, maybe the plone5 version of the documentation is in another branch? locales-future doesn't exist anymore on Plone 5, you can remove this bullet point.